### ISSUE TITLE
Maintained ?

### ISSUE THREAD

**Issue by eMerzh** [ManifestWebDesign/angular-gridster#496]
Hi, 

i was wondering if this was still maintained,  

and if not are you guys willing to pass the maintainership to someone / some group ?
not really sure i'm willing myself, but let's at least be clear  :)

Btw, i use this in one of my tool, and it's working great 👍  good job and thanks for that

**Comment by eMerzh** [comment]
ping @danomatic @davidstelter @jaimeburnap

**Comment by danomatic** [comment]
Not really. I'm busy with many other things, and mostly work in React now. However I would like to see continued bug fixes and performance updates.  If someone wants to take it over, let me know.
